pyuvdata/uvdata.py

```diff
@@ -1170,7 +1208,39 @@ def write_uvfits(self, filename, spoof_nonessential=False,
                                 force_phase=force_phase, run_check=run_check,
                                 run_check_acceptability=run_check_acceptability)
         del(uvfits_obj)
+    def read_ms(self, filepath, run_check=True,
```

Comment:
This version of the read_ms docstring is better.  Its a bit confusing that there are two versions of the same thing.  Maybe just refer one to the other?